cis-eks-controls/CIS-5.1.2.json

```diff
@@ -9,9 +9,12 @@
         "https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html#scanning-repository"
     ],
     "attributes": {
-        "armoBuiltin": true
+        "armoBuiltin": true,
+        "actionRequired": "requires review"
```

Comment:
let's not add this attribute yet as it doesn't work properly, there's a [bug open for it](https://cyberarmor-io.atlassian.net/browse/SUB-665)